### DISCUSSION TITLE
v11.0 Released! 🎉

### DISCUSSION THREAD

**Discussion by MikeCodesDotNET** [AvaloniaUI/Avalonia#12041] [General]
![v11 GA Tweet](https://github.com/AvaloniaUI/Avalonia/assets/552074/292bcf64-ee1c-4a65-a1ee-8ab7546ba16b)

Hello Avalonia Community!

We are beyond excited to announce the arrival of Avalonia 11.0! 

This monumental update embodies the culmination of countless hours of dedication and passion from our incredible community. We believe Avalonia 11 is a game-changer in the realm of cross-platform application development, providing an efficient, feature-rich, and seamless experience for creating exceptional UIs across all major platforms.

Check out our in-depth v11 release blog post to explore the wealth of features packed into this incredible new version. For those of you who are already developing with Avalonia and want to upgrade to v11, we've put together an [upgrade guide](https://docs.avaloniaui.net/docs/next/stay-up-to-date/upgrade-from-0.10) to facilitate a smooth transition.

Last but definitely not least, we would like to extend our sincere gratitude to all the contributors and sponsors who made v11 possible. Your hard work, dedication, and passion for improving and evolving Avalonia never cease to amaze us.

Here's to the next chapter of Avalonia's journey and the future of cross-platform .NET app development!

## Our wonderful sponsors: 
If you’re interested in supporting Avalonia then become a [sponsor](https://github.com/sponsors/AvaloniaUI)! 

[Jammyo](https://github.com/Jammyo)
[Terinfire](https://github.com/terinfire)
[Narafy](https://github.com/Narafy)
[Mozesa](https://github.com/mozesa)
[Doublecount](https://github.com/doublecount)
[Marcosgerene](https://github.com/marcosgerene)
[Aws](https://github.com/aws)
[Llfab](https://github.com/llfab)
[Moloneymb](https://github.com/moloneymb)
[Osamakawish](https://github.com/osamakawish)
[Amonith](https://github.com/Amonith)
[Aldelaro5](https://github.com/aldelaro5)
[Ljcollins25](https://github.com/ljcollins25)
[Filipnavara](https://github.com/filipnavara)
[Nicola36631](https://github.com/nicola36631)
[DmitryNizhebovsky](https://github.com/DmitryNizhebovsky)
[Arpadbarta](https://github.com/arpadbarta)
[Rbnswartz](https://github.com/rbnswartz)
[Florianwachs](https://github.com/florianwachs)
[Mrxx99](https://github.com/Mrxx99)
[Sandrock](https://github.com/sandrock)
[Mgnslndh](https://github.com/mgnslndh)
[KirillOsenkov](https://github.com/KirillOsenkov)
[TETYYS](https://github.com/TETYYS)
[EdWordy](https://github.com/EdWordy)
[Binkus](https://github.com/Binkus)
[Ptasev](https://github.com/ptasev)
[Saiganov](https://github.com/saiganov)
[BaseHead LLC](https://opencollective.com/steve-tushar)
[Fork](https://opencollective.com/fork)
[NFC21](https://opencollective.com/nfc21) 
[Stephen White](https://opencollective.com/stephen-white)
[Robert (roblthegreat)](https://opencollective.com/robert-roblthegreat-l)
Markus Letschka 
[Yatao Li](https://opencollective.com/yatao-li)
[Enttec](https://opencollective.com/enttec)
[NilFusion](https://opencollective.com/nilfusion)
[FMRTE](https://opencollective.com/fmrte)
[Arsen Mkrtchyan](https://opencollective.com/arsen-mkrtchyan)
[Friedrich von Never](https://opencollective.com/friedrich-von-never)
[Nitroxis](https://opencollective.com/nitroxis)
[Derek Antrican](https://opencollective.com/derek-antrican)
[Warrington Software Ltd](https://opencollective.com/warrington-software-ltd)
[Algolia](https://opencollective.com/algolia)
[Dzmitry Lahoda](https://opencollective.com/dzmitry-lahoda)
[Dorin Foaltin](https://opencollective.com/dorin-foaltin)
[Cong Zhang](https://opencollective.com/cong-zhang)
[Reuben Swartz](https://opencollective.com/reuben-swartz)
[Shynome](https://opencollective.com/shynome)
[Cisco Martinez](https://opencollective.com/cisco-martinez)
[Steve Cross](https://opencollective.com/hairlesshobo)
[Maksim Vykhota](https://opencollective.com/maksim-vykhota)
[Sergey Svirsky](https://opencollective.com/happi-cat) 
[The Mad Botter INC](https://opencollective.com/themadbotterinc)
[Oceanside Software Corporation](https://opencollective.com/oceansidesw)
[Happen Space](https://opencollective.com/happen-space)
[Dan](https://opencollective.com/dan2)
[IlI](https://opencollective.com/ili_c)
[Skitoy4321](https://opencollective.com/skitoy4321)
[KebinuChioousu](https://opencollective.com/kiera-meredith)
Martin Kramer 
[Asaf Agami](https://opencollective.com/asaf-agami)
Josua Jaeger  
[Nicolas Musset](https://opencollective.com/kryptosfr)
[Markus Wenzl](https://opencollective.com/markus-wenzl)
[Christosk92](https://opencollective.com/christosk92)
[Dong Bin](https://opencollective.com/dong-bin1)
[Mihail Kochanov](https://opencollective.com/mihail-kochanov)
[Mike Ward](https://opencollective.com/mike-ward)
[Mark Dockery](https://opencollective.com/mark-dockery)
[Markus K](https://opencollective.com/markus-k1)
[Pasoma](https://opencollective.com/pasoma)
[Christos Karapasias](https://opencollective.com/christos-karapasias)
[Henrique Nunes](https://opencollective.com/henrique-nunes)
[Aljosa Stos](https://opencollective.com/aljosa-stos)
[AF](https://opencollective.com/a-f2)
[Linrui](https://opencollective.com/linrui)
[Michael Ranciglio](https://opencollective.com/michael-ranciglio)
[Hank Grabowski](https://opencollective.com/hank-grabowski)
[Martin Pozor](https://opencollective.com/martin_pozor)
[Oxc3](https://opencollective.com/oxc3)
Bruno Garcia 
Ivan Kochurkin 
[Juan Blanco](https://opencollective.com/juan-blanco)
Denis Goriachev 
[MGage Morgan](https://opencollective.com/mgagemorgan)
[NicoVIII](https://opencollective.com/nicoviii)
Christian Findlay 
[Tyler Hasty](https://opencollective.com/tyler-hasty)
[R0ot ChiXor](https://opencollective.com/r-0-ot-chixor)
Eduardo Larios 
[XiaoLinger](https://opencollective.com/xiaolinger)
Antony Scott  
Anchisi Alexandre 
Andrei Kurosh 
[Vasily Sidorov](https://opencollective.com/bazzilic)
[Tim Ahrentløv](https://opencollective.com/tim-ahrentlov)
[Roman Bukin](https://opencollective.com/roman-bukin)
[Founntain](https://opencollective.com/founntain)
[LuoRain](https://opencollective.com/luorain)
[Finter Mobility AS](https://opencollective.com/finter-mobility-as)
[Gritworld](https://opencollective.com/gritworld)
[Gary Hertel](https://opencollective.com/gary-hertel)

**Comment by hahyes** [comment]
Congratulations! Big thanks to all contributors and sponsors! <3

**Comment by hahyes** [comment]
@MikeCodesDotNET Little question, is this demo showed in article (Ecosystem Growth part) on blog available somewhere?

**Reply by MikeCodesDotNET** [reply]
Its a sample app from [Actipro Software](https://www.actiprosoftware.com/) for Avalonia. It's not yet released, but make sure to keep an eye on their website.

**Comment by MrOldOwl** [comment]
🎉🎉🎉Congratulations on the new version. I wish all developers good health and success in future developments! Thank you.🎉🎉🎉

**Comment by zse4t7i** [comment]
Bravo!!!

**Comment by flarive** [comment]
Yeahhhh guys !

**Comment by christosk92** [comment]
Congrats guys! Big milestone!

**Comment by nielsbosma** [comment]
Congrats! Well done.

**Comment by kyuranger** [comment]
Thanks for all sponsors work.

**Comment by nnnpa31** [comment]
> [I imagine in 2050 11.0.0 Preview 246 will be released:) Just kidding. Good job guys, looking forward to RC!!!](https://github.com/AvaloniaUI/Avalonia/discussions/10729#discussioncomment-5386351)

We can't see the preview 246 that will be released in 2050 anymore :'(

Just kidding. Conguatulations! I've waited too long for this moment!
